application/common/config/main.php

```diff
@@ -201,7 +201,7 @@
                     'clientOptions' => [
                         'attach_stacktrace' => false, // stack trace identifies the logger call stack, not helpful
                         'environment' => YII_ENV,
-                        'release' => 'idp-id-broker@6.6.1',
+                        'release' => 'idp-id-broker@6.6.2',
```

Comment:
Refactoring from bool to array seems like a second digit kind of change.

Reply:
Right, it's from the perspective of the "consumer" or "user" of this API. From that perspective, it's a minor change.